pegasus/test/fixtures/fake_dashboard.rb

```diff
@@ -305,9 +305,8 @@ def self.create_fake_dashboard_db
     # Reuse the same connection in Sequel to share access to the temporary tables.
     connection = ActiveRecord::Base.connection.instance_variable_get(:@connection)
     connection.query_options[:as] = :hash
-    Sequel.extension :meta_def
-    @@fake_db = Sequel.mysql2
-    @@fake_db.meta_def(:connect) {|_| connection}
+    @@fake_db = Sequel.mysql2(test: false)
+    @@fake_db.pool.available_connections.replace([connection])
```

Comment:
`meta_def` was removed and some other parts of Sequel's connection-handling implementation were changed in the latest release, which is why this connection-stubbing test-helper implementation changed a bit.